src/pybind/mgr/cephadm/module.py

```diff
@@ -1398,10 +1398,18 @@ def remove_host(self, host: str, force: bool = False, offline: bool = False) ->
                 for d in daemons:
                     daemons_table += "{:<20} {:<15}\n".format(d.daemon_type, d.daemon_id)
 
-                return "Not allowed to remove %s from cluster. " \
+                raise OrchestratorValidationError("Not allowed to remove %s from cluster. " \
                     "The following daemons are running in the host:" \
                     "\n%s\nPlease run 'ceph orch host drain %s' to remove daemons from host" % (
-                        host, daemons_table, host)
+                        host, daemons_table, host))
+
+        # check, if there we're removing the last _admin host
+        if not force:
+            p = PlacementSpec(label='_admin')
+            if not p.filter_matching_hostspecs(self.inventory.all_specs()):
+                raise OrchestratorValidationError("You are about remove a host without having any host with the"
+                                                  "'_admin' label. Please add the '_admin' label to a host or"
+                                                  "add --force to this command")
```

Comment:
The comment at the top of this block implies we are checking we're removing the last host that has the admin label but the actual check seems to be if we have any hosts with _admin label currently (i.e. if there were 2 hosts, one with _admin label and user tries to remove _admin one, this check would pass). Do we want to also require force for actually removing the last _admin host?

Reply:
fixed